deep-learning/_posts/2017-03-16-resnet.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: review
+title: (ResNet) Deep Residual Learning for Image Recognition
```

Comment:
I think that "ResNet : Deep Residual Learning..." would be more readable than using parentheses. If you use a colon you must surround the title with quotes. (not a big deal)